### ISSUE TITLE
[Feature request] Add manually car power consumption 

### ISSUE THREAD

**Issue by iot-sle** [solectrus/solectrus#1625]
Senec don't offer the car power consumption in exported statistic values. But we can get the past values per day from the Android/IOS app. There is no interface to get those past values into Solectrus. So an imput box for manually add those values would be great. 

(b) The total engergy consumption of the vehicle(s) is of interest too.
It may be useful to add also  power consumption from public wallboxes and an identifier for the car manually. I know that (b) is much more work and adaption in interfaces

**Comment by ledermann** [comment]
Yes, wallbox measurements are not included in the CSV files that SENEC provides for download on mein-senec.de. However, they are included in the data that is transmitted to SENEC every five minutes, which can be viewed on SENEC's mobile apps or website. While it would be possible to build additional data entry forms to manually copy the measurements to SOLECTRUS, I doubt that many people will do this.

Recording charging processes at public charging stations seems like a neat idea. I will think about it.

But beware: The wallbox does not differentiate between cars, so you never know **which** car was charged.
